pkg/utils/config.go

```diff
@@ -0,0 +1,164 @@
+package utils
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/satori/go.uuid"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"os"
+	"os/user"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"text/template"
+)
+
+var defaultEnvConfig = `#generated by eden
+adam-dist: {{ .Dist }}/adam
+image-dist: {{ .Dist }}/images
+downloader-dist: {{ .Dist }}/eve/dist/{{ .Arch }}
+downloader-save: true
+eserver-port: 8888
+eserver-pid: {{ .Dist }}/eserver.pid
+eserver-log: {{ .Dist }}/eserver.log
+eve-arch: {{ .Arch }}
+eve-os: {{ .OS }}
+eve-accel: true
+adam-port: 3333
+eve-serial: 31415926
+eve-cert: {{ .Dist }}/certs/onboard.cert.pem
+eve-config: {{ .Dist }}/qemu.conf
+eve-pid: {{ .Dist }}/eve.pid
+eve-log: {{ .Dist }}/eve.log
+adam-force: true
+certs-dist: {{ .Dist }}/certs
+domain: mydomain.adam
+ip: {{ .IP }}
+uuid: {{ .UUID }}
+hv: kvm
+eve-firmware: {{ .Dist }}/eve/dist/amd64/OVMF.fd
+image-file: {{ .Dist }}/eve/dist/amd64/live.qcow2
```

Comment:
Question about the image file: am I correct that if I point it at a file that already exists when I run any of the eden commands it will simply be used and not touched?

Reply:
Yes, it will not touched, if exists. 
Also, I move image file flag from the qemu.conf file to args of qemurun command:
https://github.com/lf-edge/eden/blob/ea676f293e39ed98b8fd87103d1a4880316554e5/cmd/qemurun.go#L108